Checklist

```diff
@@ -0,0 +1,24 @@
+<!-- Check all that apply [x] -->
+
+# Checklist
+
+This is a checklist that can be used in effort to review before submitting a PR, in the Edge Drivers Repo.
+
+- [ ] This is the first time I am contributing code:
+     - [ ] I have reviewed the README.md file
+     - [ ] I have reviewed the CODE_OF_CONDUCT.md file
+     - [ ] I have signed the CLA
+     - [ ] This is a request to add a device for WWST certification and I plan on entering a console request or have entered a console request at developer.smartthings.com
```

Comment:
The info in this bullet, could be included in the wwst cert request bullet under "Type of Change"

Reply:
Done